rmgpy/data/kinetics/family.py

```diff
@@ -1067,8 +1065,7 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
         # If product structures are Molecule objects, update their atom types
         for struct in productStructures:
             if isinstance(struct, Molecule):
-                struct.updateAtomTypes()
-                struct.updateMultiplicity()
+                struct.update()
```

Comment:
if it's *not* a `Molecule` then `updateConnectivityValues` is no longer called (removed from line 1046 above). What will this effect and does it matter? what could it be besides a `Molecule`? a `Group` maybe? (eg. in a reaction template?)